### ISSUE TITLE
FullPageOS 0.6.0 RC1

### ISSUE THREAD

**Issue by guysoft** [guysoft/FullPageOS#18]
Hey,
The long delays FullPageOS 0.6 first release candidate.
The image should work on the new RaspberryPi 3, it takes up half of the size, and packs an up to date chromium browser with a new tool that lets you add multiple tabs.

Would be nice if people can confirm it works for them, or not:

Image:
http://docstech.net/FullPageOS/2016-05-27_2016-05-10-fullpageos-jessie-lite-0.6.0.zip
Md5:
http://docstech.net/FullPageOS/2016-05-27_2016-05-10-fullpageos-jessie-lite-0.6.0.zip.md5

**Comment by guysoft** [comment]
Hey guys,
I can see there is a lot of traffic from [Hacker news](https://news.ycombinator.com/item?id=11809911. It would be nice if any of you that downloaded can report back here if this works well or bad for you. I am only one person and my QA relies on you.

**Comment by guysoft** [comment]
Update from @tailorvj that the vnc password doea not work. Would be nice if anyone can confirm. If this is the case expect RC3

**Comment by Dr-Juice** [comment]
I've installed it, and we're using it to display dashboards in our office.  Right now, we have a Pi 3 running FullPageOS to display the status page of our Observium environment.  I'm ordering 2 more Pi 3's to run as FullPageOS kiosks in other departments to display web pages from SQL reports in our material testing lab.

The issues I've found so far are, the VNX password isn't 'raspberry', and are there instructions on how to configure and use the FullPageDashboard app?

**Comment by guysoft** [comment]
@Dr-Juice The VNC password issue is here, including a workaround #26 
There are no instructions for FullPageDashbaord, it was written by @amitdar

**Comment by tailorvj** [comment]
@guysoft what's our schedule for 0.60 RC2 (or is it 0.61)?

**Comment by guysoft** [comment]
I am going tonight to a Burning Man regional event for a week (Midburn)
after months of planning. So not untill that is over.
There is a build server, so I can release builds from my phone, but not
commit any code.
I suggest if someone wants FullPageOS to develop faster, then come to aid.
This is a project that I developed because I saw a need, and I think this
week with it going viral we saw its the case. But I can't test it and
maintain it myself.
I have my hands full with my startup and OctoPi already.
I will gladly teach people how to build it, and where to edit, its pretty
simple.
